metrics/metrics.go

```diff
@@ -0,0 +1,41 @@
+package metrics
+
+import (
+	"encoding/json"
+	"io"
+
+	metrics "github.com/rcrowley/go-metrics"
+)
+
+type Metrics struct {
```

Comment:
__[golint]__
* [golint] exported type Metrics should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->